src/acrquery/azext_acrquery/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67"
```

Comment:
Are you sure the `azext.minCliCoreVersion` is `2.0.67`?

Reply:
updated